src/interfaces/resource-interfaces/resource-interface.ts

```diff
@@ -82,4 +82,23 @@ export interface RegistrationResourceIF {
   }
 }
 
-export interface ResourceIF extends DissolutionResourceIF, IncorporationResourceIF, RegistrationResourceIF {}
+/** Restoration resource interface. */
+export interface RestorationResourceIF {
+  // *** TODO: update
```

Comment:
I think this is as far as I'll go in this PR here. Future tickets can update this interface as needed.